.github/workflows/submodule_update.yml

```diff
@@ -0,0 +1,32 @@
+name: Submodule_update
+
+on:
+  push:
+    branches: 
+      - 'master'
+      - '2.10'
+
+jobs:
+  tarantool:
+    name: Update submodule in ee repository
+    runs-on: ubuntu-20.04-self-hosted
+    env:
+      SUBMODULE: 'tarantool'
+      REPOSITORY: "${{ secrets.EE_UPDATE_SUBMODULE_REPO }}"
+      FEATURE_BRANCH: 'TarantoolBot/update-submodule'
```

Comment:
It should be `TarantoolBot/update-submodule-${{ github.ref_name }}`

Reply:
@ylobankov what about `.../update-submodule-tarantool-${{ github.ref_name }}`?

Reply:
In a private discussion you suggested `update-tarantool-${{ github.ref_name }}`, I am OK with it.